blueprints/@gavant/ember-bootstrap-dropdown/index.js

```diff
@@ -9,33 +9,28 @@ module.exports = {
     normalizeEntityName() {},
 
     afterInstall() {
-        let importStatement =
-            '\n@import "ember-basic-dropdown";\n' +
-            '@import "gavant-ember-bootstrap-dropdown";\n';
+        let importStatement = '\n@import "ember-basic-dropdown";\n' + '@import "gavant-ember-bootstrap-dropdown";\n';
 
         let stylePath = path.join('app', 'styles');
         let file = path.join(stylePath, 'app.scss');
         let writeOp = Promise.resolve();
 
-        if(!fs.existsSync(stylePath)) {
+        if (!fs.existsSync(stylePath)) {
             fs.mkdirSync(stylePath);
         }
 
-        if(fs.existsSync(file)) {
+        if (fs.existsSync(file)) {
             this.ui.writeLine(`Added import statement to ${file}`);
             writeOp = this.insertIntoFile(file, importStatement, {});
         } else {
             fs.writeFileSync(file, importStatement);
             this.ui.writeLine(`Created ${file}`);
         }
 
-        return writeOp
-            .then(() => {
-                return this.addAddonsToProject({
-                    packages: [
-                        {name: 'ember-basic-dropdown', target: '^3.0.6'}
-                    ]
-                });
+        return writeOp.then(() => {
+            return this.addAddonsToProject({
+                packages: [{ name: 'ember-basic-dropdown', target: '^3.0.6' }]
```

Comment:
looks like we are using `5.0.0` in the internal deps, but still installing `3.0.6` in the consuming app here.